topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.md

```diff
@@ -0,0 +1,1778 @@
+>        - *"Number to show"*: `10`
+>        - *"Variable(s) to regress out"*: `percent.mt`
+>
+> 2. Rename the output as `Preprocessed Data`
+>
+>    If we inspect this dataset, we can see that a new assay called `SCT` has been created and is set as the default assay. Click on the {% icon galaxy-eye %}- of the following output to check this.
+>
+> 3. {% tool [Seurat Data Management](toolshed.g2.bx.psu.edu/repos/iuc/seurat_data/seurat_data/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Preprocessed Data` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Inspect Seurat Object`
+>        - *"Display information about"*: `General`
+>
+{: .hands_on}
+</div>
+
+### Visualise Highly Variable Genes
+
+><hands-on-title>Visualise Results</hands-on-title>
+>
+> 1. {% tool [Seurat Visualize](toolshed.g2.bx.psu.edu/repos/iuc/seurat_plot/seurat_plot/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `Preprocessed Data` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Plot Variable Genes with 'VariableFeaturePlot'`
+>        - *"Label the top most variable features"*: `Yes`
+>        - *"Number to show"*: `10`
+>
+{: .hands_on}
+
+<div class='Separate-Preprocessing-Steps' markdown='1'>
+
+![Most genes are grouped together at the bottom of the plot with low standardized variance. The selected variable genes are highlighted in red and labelled by name. They have higher standardized variances although most are still quite close to the non variable genes. The labels read PPBP, S100A9, LYZ, IGLL5, GNLY, FTL, PF4, FTH1, GNG11, and FCER1A.](../../images/scrna-seurat-pbmc3k/seurat_variable_genes.png "Plot showing the standardized variances of variable and non-variable genes. The top 10 most variable genes are labelled.")
+
+</div>
+
+<div class='SCTransform' markdown='1'>
+
+![Most genes are closely grouped together, right at the bottom of the plot with low standardized variance. The selected variable genes are highlighted in red and labelled by name. They are higher up on the plot, showing they have higher standardized variances although most are still quite close to the non variable genes. They are S100A9, GNLY, LYZ, S100A8, NKG7, FTL, GZMB, IGLL5, FTH1, and CCL5.](../../images/scrna-seurat-pbmc3k/seurat_variable_genes_SCT.png "Plot showing the standardized variances of variable and non-variable genes. The top 10 most variable genes are labelled.")
+
+</div>
+
+> <question-title></question-title>
+> 1. What are the top 10 most variable genes in this dataset?
+> 2. Why are we interested in these genes?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. We can check the list of the top variable genes in our history by clicking on the {% icon galaxy-eye %} or see them on our variable features plot.
+> > <div class='Separate-Preprocessing-Steps' markdown="1">
```

Comment:
The CYOA solutions were broken because you used `<span>` elements, I changed it to `<div>` with `markdown="1"` and seems like it works now.

I know this appears in a couple more places, so please update throughout :)